Update scripts and languages for upstream changes (Unicode 9ish)

From https://github.com/harfbuzz/icu-le-hb/commit/01b1cb97a742662389a64f2e922bdd2ecbbea52d

See also https://github.com/unicode-org/icu/commit/997ed6a7cdd7ca5f0111ba887a1ad376006f14af
--- src/LELanguages.h.orig
+++ src/LELanguages.h
@@ -1,11 +1,11 @@
 /*
  *
- * (C) Copyright IBM Corp. 1998-2015. All Rights Reserved.
+ * (C) Copyright IBM Corp. 1998-2016. All Rights Reserved.
  *
  * WARNING: THIS FILE IS MACHINE GENERATED. DO NOT HAND EDIT IT UNLESS
  * YOU REALLY KNOW WHAT YOU'RE DOING.
  *
- * Generated on: 04/27/2015 10:39:37 AM PDT
+ * Generated on: 04/08/2016 02:00:11 PM PDT
  */
 
 #ifndef __LELANGUAGES_H
--- src/LEScripts.h.orig
+++ src/LEScripts.h
@@ -1,11 +1,11 @@
 /*
  *
- * (C) Copyright IBM Corp. 1998-2014. All Rights Reserved.
+ * (C) Copyright IBM Corp. 1998-2016. All Rights Reserved.
  *
  * WARNING: THIS FILE IS MACHINE GENERATED. DO NOT HAND EDIT IT UNLESS
  * YOU REALLY KNOW WHAT YOU'RE DOING.
  *
- * Generated on: 04/08/2014 03:20:04 PM PDT
+ * Generated on: 04/08/2016 02:00:11 PM PDT
  */
 
 #ifndef __LESCRIPTS_H
@@ -69,24 +69,12 @@ enum ScriptCodes {
     thaaScriptCode = 37,
     thaiScriptCode = 38,
     tibtScriptCode = 39,
-/**
- * @deprecated ICU 54. See {@link icu::LayoutEngine}
- */
-
     cansScriptCode = 40,
-/**
- * @deprecated ICU 54. See {@link icu::LayoutEngine}
- */
-
     yiiiScriptCode = 41,
     tglgScriptCode = 42,
     hanoScriptCode = 43,
     buhdScriptCode = 44,
     tagbScriptCode = 45,
-/**
- * @deprecated ICU 54. See {@link icu::LayoutEngine}
- */
-
     braiScriptCode = 46,
     cprtScriptCode = 47,
     limbScriptCode = 48,
@@ -95,15 +83,7 @@ enum ScriptCodes {
     shawScriptCode = 51,
     taleScriptCode = 52,
     ugarScriptCode = 53,
-/**
- * @deprecated ICU 54. See {@link icu::LayoutEngine}
- */
-
     hrktScriptCode = 54,
-/**
- * @deprecated ICU 54. See {@link icu::LayoutEngine}
- */
-
     bugiScriptCode = 55,
     glagScriptCode = 56,
     kharScriptCode = 57,
@@ -111,10 +91,6 @@ enum ScriptCodes {
     taluScriptCode = 59,
     tfngScriptCode = 60,
     xpeoScriptCode = 61,
-/**
- * @deprecated ICU 54. See {@link icu::LayoutEngine}
- */
-
     baliScriptCode = 62,
     batkScriptCode = 63,
     blisScriptCode = 64,
@@ -157,10 +133,6 @@ enum ScriptCodes {
     xsuxScriptCode = 101,
     zxxxScriptCode = 102,
     zzzzScriptCode = 103,
-/**
- * @deprecated ICU 54. See {@link icu::LayoutEngine}
- */
-
     cariScriptCode = 104,
     jpanScriptCode = 105,
     lanaScriptCode = 106,
@@ -173,10 +145,6 @@ enum ScriptCodes {
     sundScriptCode = 113,
     moonScriptCode = 114,
     mteiScriptCode = 115,
-/**
- * @deprecated ICU 54. See {@link icu::LayoutEngine}
- */
-
     armiScriptCode = 116,
     avstScriptCode = 117,
     cakmScriptCode = 118,
@@ -191,18 +159,10 @@ enum ScriptCodes {
     tavtScriptCode = 127,
     zmthScriptCode = 128,
     zsymScriptCode = 129,
-/**
- * @deprecated ICU 54. See {@link icu::LayoutEngine}
- */
-
     bamuScriptCode = 130,
     lisuScriptCode = 131,
     nkgbScriptCode = 132,
     sarbScriptCode = 133,
-/**
- * @deprecated ICU 54. See {@link icu::LayoutEngine}
- */
-
     bassScriptCode = 134,
     duplScriptCode = 135,
     elbaScriptCode = 136,
@@ -216,10 +176,6 @@ enum ScriptCodes {
     palmScriptCode = 144,
     sindScriptCode = 145,
     waraScriptCode = 146,
-/**
- * @deprecated ICU 54. See {@link icu::LayoutEngine}
- */
-
     afakScriptCode = 147,
     jurcScriptCode = 148,
     mrooScriptCode = 149,
@@ -229,34 +185,27 @@ enum ScriptCodes {
     takrScriptCode = 153,
     tangScriptCode = 154,
     woleScriptCode = 155,
-/**
- * @deprecated ICU 54. See {@link icu::LayoutEngine}
- */
-
     hluwScriptCode = 156,
     khojScriptCode = 157,
     tirhScriptCode = 158,
-/**
- * @deprecated ICU 54. See {@link icu::LayoutEngine}
- */
-
     aghbScriptCode = 159,
     mahjScriptCode = 160,
-/**
- * @deprecated ICU 54. See {@link icu::LayoutEngine}
- */
-
     ahomScriptCode = 161,
     hatrScriptCode = 162,
     modiScriptCode = 163,
     multScriptCode = 164,
     paucScriptCode = 165,
     siddScriptCode = 166,
-
-/**
- * @deprecated ICU 54. See {@link icu::LayoutEngine}
- */
-    scriptCodeCount
+    adlmScriptCode = 167,
+    bhksScriptCode = 168,
+    marcScriptCode = 169,
+    newaScriptCode = 170,
+    osgeScriptCode = 171,
+    hanbScriptCode = 172,
+    jamoScriptCode = 173,
+    zsyeScriptCode = 174,
+
+    scriptCodeCount = 175
 };
 
 U_NAMESPACE_END
--- src/ScriptAndLanguageTags.cpp.orig
+++ src/ScriptAndLanguageTags.cpp
@@ -1,11 +1,11 @@
 /*
  *
- * (C) Copyright IBM Corp. 1998-2015. All Rights Reserved.
+ * (C) Copyright IBM Corp. 1998-2016. All Rights Reserved.
  *
  * WARNING: THIS FILE IS MACHINE GENERATED. DO NOT HAND EDIT IT UNLESS
  * YOU REALLY KNOW WHAT YOU'RE DOING.
  *
- * Generated on: 04/27/2015 10:39:37 AM PDT
+ * Generated on: 04/08/2016 02:00:11 PM PDT
  */
 
 #include "LETypes.h"
@@ -181,7 +181,15 @@ const LETag OpenTypeLayoutEngine::scriptTags[] = {
     modiScriptTag, /* 'modi' (MODI) */
     multScriptTag, /* 'mult' (MULTANI) */
     paucScriptTag, /* 'pauc' (PAU_CIN_HAU) */
-    siddScriptTag  /* 'sidd' (SIDDHAM) */
+    siddScriptTag, /* 'sidd' (SIDDHAM) */
+    adlmScriptTag, /* 'adlm' (ADLAM) */
+    bhksScriptTag, /* 'bhks' (BHAIKSUKI) */
+    marcScriptTag, /* 'marc' (MARCHEN) */
+    newaScriptTag, /* 'newa' (NEWA) */
+    osgeScriptTag, /* 'osge' (OSAGE) */
+    hanbScriptTag, /* 'hanb' (HANB) */
+    jamoScriptTag, /* 'jamo' (JAMO) */
+    zsyeScriptTag  /* 'zsye' (ZSYE) */
 };
 
 const LETag OpenTypeLayoutEngine::languageTags[] = {
--- src/ScriptAndLanguageTags.h.orig
+++ src/ScriptAndLanguageTags.h
@@ -1,11 +1,11 @@
 /*
  *
- * (C) Copyright IBM Corp. 1998-2015. All Rights Reserved.
+ * (C) Copyright IBM Corp. 1998-2016. All Rights Reserved.
  *
  * WARNING: THIS FILE IS MACHINE GENERATED. DO NOT HAND EDIT IT UNLESS
  * YOU REALLY KNOW WHAT YOU'RE DOING.
  *
- * Generated on: 04/27/2015 10:39:37 AM PDT
+ * Generated on: 04/08/2016 02:00:11 PM PDT
  */
 
 #ifndef __SCRIPTANDLANGUAGES_H
@@ -196,6 +196,14 @@ const LETag modiScriptTag = 0x6D6F6469; /* 'modi' (MODI) */
 const LETag multScriptTag = 0x6D756C74; /* 'mult' (MULTANI) */
 const LETag paucScriptTag = 0x70617563; /* 'pauc' (PAU_CIN_HAU) */
 const LETag siddScriptTag = 0x73696464; /* 'sidd' (SIDDHAM) */
+const LETag adlmScriptTag = 0x61646C6D; /* 'adlm' (ADLAM) */
+const LETag bhksScriptTag = 0x62686B73; /* 'bhks' (BHAIKSUKI) */
+const LETag marcScriptTag = 0x6D617263; /* 'marc' (MARCHEN) */
+const LETag newaScriptTag = 0x6E657761; /* 'newa' (NEWA) */
+const LETag osgeScriptTag = 0x6F736765; /* 'osge' (OSAGE) */
+const LETag hanbScriptTag = 0x68616E62; /* 'hanb' (HANB) */
+const LETag jamoScriptTag = 0x6A616D6F; /* 'jamo' (JAMO) */
+const LETag zsyeScriptTag = 0x7A737965; /* 'zsye' (ZSYE) */
 
 const LETag nullScriptTag = 0x00000000; /* ''     (NULL) */
 
